source/blog/2017-01-23-ember-2-11-released.md

```diff
@@ -0,0 +1,372 @@
+likely issues, please report back with any problems you discover in your private
+addons.
+
+#### Performance
+
+One of the major themes for the past six weeks has been a focus on the build
+performance of Ember CLI. We've added instrumentation to understand where we're
+spending time and have begun the consistent incremental work required to bring
+build times down without changing the build output. This has been a team effort
+with contributions from David Hamilton, Robert Jackson, Stefan Penner, Trent
+Willis, and more.
+
+#### Developers, Developers, Developers, Developers
+
+We've made a tremendous number of behind-the-scene changes to improve the
+ergonomics for contributing to Ember CLI. This leverage is how we achieve
+synergy across all contributors in order to deliver measurable results.
+
+- We've adopted ESLint for Ember CLI itself, painted a few bikesheds, and
+now have a much more consistent codebase. PR comments around code style should
+come with an associated style rule change to enforce that behavior so that it
+is consistent given the multitude of reviewers we have.
+- Now that we have dropped support for Node.js 0.12 we have begun adoption of
+all of the nice things we weren't previously able to use; most-specifically
+**ES6 classes**! We've modified `core-object` as well in order to make it
+compatible with ES6 classes.
+- We've adopted Yarn for development of Ember CLI. We now use it for CI and are
+using this to lay a foundation for making `yarn` a default and _supported_
+option for package management in applications.
+- By virtue of a lot of work in CI and on caching test run times have been
+reduced to seven minutes. Check out [PackageCache](https://github.com/ember-cli/ember-cli/blob/master/tests/helpers/package-cache.js)
+the piece which got us most of the reward.
+- We've introduced an experiments API which allows us to incrementally land
+invasive changes behind an experiment flag. The experiment flag prevents their
+use except on the `canary` branch. This allows us to experiment with API shape
+and understand problems without committing to publishing a feature. Currently
+the instrumentation changes are behind an experiment flag which allows us to
+understand the needs and write a [much more complete RFC](https://github.com/ember-cli/rfcs/pull/90).
+
+We hope to continue making changes that make it easier to contribute to Ember
+CLI with higher confidence.
+
+#### Other Notable Changes
+
+- `ember new` and `ember addon` correctly support the `--directory` argument and
+allow you to specify an existing empty directory.
+- Removed `ember-data` and `ember-cli-app-version` from the default
+`ember addon` blueprint.
+- Duplicate calls to `.import('assetname.js)'` will no longer include the asset
+in the output file twice.
+- Given that we no longer require `bower` in the default scenario, we now lazily
+install it into your application the first time that you need it.
```

Comment:
@nathanhammond we should add here that global NPM will now be used and requires you to have NPM 3+ installed